apps/src/applab/redux/applab.js

```diff
@@ -44,10 +72,34 @@ function interfaceMode(state, action) {
   }
 }
 
+function redirectDisplay(state, action) {
```

Comment:
Add a comment explaining what this function does.

Reply:
Woops. I don't think I made a clear comment earlier. I was suggesting you add a comment to the function header (similar to dismissRedirectNotice earlier). That being said, if it's not normal for us to do that in reducers, then nevermind :)

Reply:
I checked out some other reducers and it doesn't look like we have done this in the past. I'm hoping the inner comments make the purpose of this reducer clear, but similar to the initialization, I think it's worth keeping this in mind to apply at a broader scale.